.github/workflows/nightly.yml

```diff
@@ -66,3 +66,33 @@ jobs:
       with:
         name: dist
         path: dist
+
+    - name: Send email on failure
+      if: failure()
+      uses: ./.github/actions/send-email
+      with:
+        api-key: ${{ secrets.OSS_BOT_MAILGUN_KEY }}
+        domain: ${{ secrets.OSS_BOT_MAILGUN_DOMAIN }}
+        from: 'GitHub <admin-github@${{ secrets.OSS_BOT_MAILGUN_DOMAIN }}>'
+        to: ${{ secrets.FIREBASE_ADMIN_GITHUB_EMAIL }}
+        subject: '[${{github.repository}}] Nightly build failed!'
+        html: >
+          <b>Nightly workflow failed on: ${{github.repository}}</b>
+          <br /><br />Checkout the 
+          <a href="github.com/${{github.repository}}/actions/workflows/nightly.yml">failed workflow</a>.
```

Comment:
Let's link to https://github.com/firebase/firebase-admin-node/actions. Both here and below.